analysis_runner/README.md

```diff
@@ -16,7 +20,9 @@ analysis-runner \
     main.py and some arguments # becomes path/to/script/main.py and some arguments
 ```
 
-1. Providing the `--repository` parameter, making the script path relative to the git repository is disabled, and you must provide a commit hash too. For example:
+- **Providing** the `--repository` parameter: making the script path relative to
+  the git repository is disabled, and you must provide a commit hash too. For
+  example:
```

Comment:
```suggestion
- **Providing** the `--repository` parameter: you must specify the complete relative path from the git root directory (determining this the relative to
  the git repository is disabled), and provide a commit hash too. For
  example:
```

Another small change to make this clearer, because my initial wording was pretty average.

Reply:
```suggestion
- **Providing** the `--repository` parameter: you must specify the complete relative path from the git root directory (determining this relative to the git repository is disabled), and provide a commit hash too. For
  example:
```